### PR TITLE
Task/add config option persist errors

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/fiware-cygnus/issues/1928
- [x] update doc https://fiware-cygnus.readthedocs.io/en/2.0.0/cygnus-common/installation_and_administration_guide/error_persistance/index.html

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
+[cygnus-common][SQLBackendImpl] Add configuration option to persist errors in SQL sinks (#1928)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Similar to the existing functionality for PG?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE:1]: just make it optional by config

Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: Ok. NTC

File: cygnus-ngsi/conf/agent_ngsi.conf.template [hunk 2]
```diff
@@ -197,7 +197,8 @@ cygnus-ngsi.sinks.postgresql-sink.type = com.telefonica.iot.cygnus.sinks.NGSIPos
 #cygnus-ngsi.sinks.postgresql-sink.batch_timeout = 30
 # number of retries upon persistence error
 #cygnus-ngsi.sinks.postgresql-sink.batch_ttl = 10
-
+# true if errors are persisted for this sink, false otherwise
+#cygnus-ngsi.sinks.postgresql-sink.persist_errors = true
```
Comment (fgalan) [inline cygnus-ngsi/conf/agent_ngsi.conf.template:201]: I think there are per-sink docu tables with the allowed parameters. Maybe persiste_errors should be added to all them.

**Review by fgalan — APPROVED** [review]
LGTM

**Review by IvanHdzC — APPROVED** [review]
LGTM

**Pull request merged**